libraries/AP_Compass/AP_Compass.h

```diff
@@ -310,10 +305,10 @@ friend class AP_Compass_Backend;
     // HIL variables
     struct {
         Vector3f Bearth;
```

Comment:
why keeping this ? That is unused

Reply:
Yeah, I was a bit confused by that.  I removed it then noted that the SITL backend initialised it...

I *think* you're right, and I did already remove it locally.

Thanks.

Reply:
Removed.